contracts/core/diaspore/cosigner/CollateralAuction.sol

```diff
@@ -285,7 +285,7 @@ contract CollateralAuction is ReentrancyGuard, Ownable {
             After all the `fromToken` is being offered, the auction switches and the requested
             `baseToken` goes down linearly, until reaching 1 after 24 hours
 
-        @dev If the auction is not taken after the requesting amount reaches 1, the second part
+        @dev If the auction is not taken after the requesting amount can reaches 1, the second part
```

Comment:
@dev If the auction is not taken after the minimum requesting amount, the second part